install.sh

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash
+
+echo "This will install ATHEME, IRCD-SEVEN (charybdis) AND GMS!"
+echo
+echo "Read now the README"
+
+echo
+
+curl "https://raw.githubusercontent.com/freenode/gms/master/README" | more 
+
+echo
+echo This script does all the installation described in the README in an automated way.
+echo But be sure to have read the \"Before anything else\" section.
+echo
+
+read -s -p "To continue, press enter. Otherwise, press CTRL+C" 
+echo
+
+echo Downloading and installing requirements...
+
+# Installing requirements
+sudo apt-get update
+sudo apt-get install build-essential gcc flex bison libexpat-dev libpq-dev
+sudo apt-get install perl=5.18.2-2ubuntu1
+
+wget -O- http://cpanmin.us | perl - -l ~/perl5 App::cpanminus local::lib
+eval `perl -I ~/perl5/lib/perl5 -Mlocal::lib`
+
+echo "eval \`perl -I ~/perl5/lib/perl5 -Mlocal::lib\`" >> $HOME/.bashrc
+
+cpanm --installdeps .
+
+read -s -p "Continue with ENTER"
+echo
+
+# Download all
+cd $HOME
+mkdir freenode
+cd freenode
+mkdir src
+mkdir atheme
+mkdir ircd
+cd src
+git clone https://github.com/atheme/atheme.git
+git clone https://github.com/atheme/charybdis.git ircd-seven
+git clone https://github.com/decontamin4t0R/gms.git
```

Comment:
uhhh... sorry was a mistake which has sneaked in... i setted it to this for testing and didnt remove it...